portfolio/src/main/webapp/script.js

```diff
@@ -57,16 +57,23 @@ const getComments = function() {
     fetch('/data').then(response => response.json()).then((comments) => {
         const commentsContainer = document.getElementById('comments-container');
         commentsContainer.innerHTML = '';
-        
-        for(let i = 0; i < comments.length; i++) {
-            commentsContainer.appendChild(createListElement('Comment: ' + comments[i]));
-        }
+        comments.forEach((Comment) => {
```

Comment:
`Comment` should not be capitalized since it is a variable name.

I recommend against the Array.prototype.forEach pattern - it's inverted, so it's harder to read at first glance than a for-of loop (I know I keep bringing for-of loops up, but it was created to fix all the problems with forEach, for-in, and for (let i = 0; i < ...)), and it's harder to debug since it takes in a function as an argument so you can't step directly into it in the JS debugger.